bcipy/acquisition/__init__.py

```diff
@@ -0,0 +1,8 @@
+from bcipy.acquisition.protocols.lsl.lsl_client import LslAcquisitionClient
+from bcipy.acquisition.datastream.lsl_server import LslDataServer, await_start
+
+__all__ = [
+    'LslAcquisitionClient',
+    'LslDataServer',
+    'await_start'

```

Comment:
Seeing `await_start` as one of the primary exports of the acquisition module makes me think that it should   be a server method or live elsewhere (in a threading helper?). I'll make a note to revisit this.